### DISCUSSION TITLE
Lens and several mounts

### DISCUSSION THREAD

**Discussion by bob2204** [lensfun/lensfun#2005] [Q&A] [answered]
Hello,

the SIGMA 85 DG DN correction parameters are for de SONY mount. What about the LEICA L mount ? would the correction parameters be the same and simply add the LEICA L mount in the database ?

Regards

**Comment by tuxfanx** [comment, marked as answer]
Yes, adding the mount for Leica L should be ok. I have just added the entry to the database.

**Reply by bob2204** [reply]
Cool, thanks !
